src/v2/Apps/Artist/Routes/WorksForSale/Utils/__tests__/merchandisingTrial.jest.ts

```diff
@@ -0,0 +1,55 @@
+import { useSystemContext } from "v2/System/useSystemContext"
+import {
+  MANUAL_GRID_CURATION_FEATURE_FLAG_KEY,
+  useCheckManualGridCuration,
+} from "../merchandisingTrial"
+
+jest.mock("v2/System/useSystemContext")
+
+describe("useCheckManualGridCuration", () => {
+  const mockUseSystemContext = useSystemContext as jest.Mock
+
+  describe("when the feature flag is enabled", () => {
+    beforeEach(() => {
+      mockUseSystemContext.mockImplementation(() => ({
+        featureFlags: {
+          [MANUAL_GRID_CURATION_FEATURE_FLAG_KEY]: true,
```

Comment:
Doesn't affect your implementation, as we will always return out a boolean for the `useFeatureFlag` hook, but want to flag that with the implementation of `useFeatureVariant` the shape of the`featureFlags` value in `systemContext` was changed, which you can see [here](https://github.com/artsy/force/blob/main/src%2Fv2%2FSystem%2F__tests__%2FuseFeatureFlag.jest.tsx#L11).

Reply:
Actually, take this back. See comment below.

Reply:
Updated